sklearn/metrics/_plot/roc_curve.py

```diff
@@ -118,11 +142,37 @@ def plot(self, ax=None, *, name=None, **kwargs):
 
         line_kwargs.update(**kwargs)
 
+        chance_level_line_kwargs = {
+            "label": "Chance level (AUC = 0.5)",
+            "color": "k",
+            "linestyle": "--",
+        }
+
+        chance_level_line_kwargs.update(**chance_level_kwargs)
```

Comment:
and then update it here. We can do this update `if chance_level_kwargs is not None`.